server/database/repositories/TournamentRepository.ts

```diff
@@ -41,6 +41,7 @@ function toTournamentDb(t: TournamentWithoutMS) : TournamentDb {
     ownerId: t.ownerId,
     teams: t.teams.map((team) => toTeamDb(team)),
     matches: t.matches.map((match) => toMatchDb(match)),
+    isFinished: t.isFinished,
```

Comment:
You add a useless object to Tournament. `isFinished` is always false. I think this should be another task. 
Anyway now you have to add logic to updateMatchScore to update tournament.isFinished

Reply:
No, it is not useless. We discussed it many times before. It stays here as it is.

Reply:
I mean this object doesn't have logic. In updateMatchScore You should code condition to set isFinished